pkg/relayer/session/sessiontree.go

```diff
@@ -206,3 +208,17 @@ func (st *sessionTree) Delete() error {
 	// Delete the KVStore from disk
 	return os.RemoveAll(filepath.Dir(st.storePath))
 }
+
+// MarkAsClaimed marks the session tree as claimed
+// It returns an error if it has already been marked as claimed.
+func (st *sessionTree) MarkAsClaimed() error {
```

Comment:
I think this should be called `Claim` or `StartClaiming`. Wdyt?

Reply:
`StartClaiming` is perfect